modules/map/hdmap/adapter/xml_parser/lanes_xml_parser.h

```diff
@@ -50,8 +50,18 @@ class LanesXmlParser {
   static Status ParseSpeed(const tinyxml2::XMLElement& xml_node, PbLane* lane);
   static Status ParseSampleAssociates(const tinyxml2::XMLElement& xml_node,
                                       PbLane* lane);
-  static Status ParseRoadSampleAssociates(const tinyxml2::XMLElement& xml_node,
-                                          PbLane* lane);
+  static Status ParseSingleSideRoadSampleAssociates(
+                                    const tinyxml2::XMLElement& xml_node,
+                                    PbLane* lane, bool bleft);
+  static Status ParseLeftRoadSampleAssociates(
+                                    const tinyxml2::XMLElement& xml_node,
+                                    PbLane* lane);
```

Comment:
Try `scripts/clang-format.sh` to fix the style issues. Several indents are wrong in the pull request.

Reply:
I have fixed the style issues.